eng/Versions.props

```diff
@@ -262,8 +262,8 @@
     <MicrosoftExtensionsLoggingVersion>3.1.7</MicrosoftExtensionsLoggingVersion>
     <MicrosoftSymbolStoreVersion>1.0.406601</MicrosoftSymbolStoreVersion>
     <!-- sdk version, for testing workloads -->
-    <SdkVersionForWorkloadTesting>$(MicrosoftDotNetApiCompatTaskVersion)</SdkVersionForWorkloadTesting>
-    <!--<SdkVersionForWorkloadTesting>9.0.102</SdkVersionForWorkloadTesting>-->
+    <!-- <SdkVersionForWorkloadTesting>$(MicrosoftDotNetApiCompatTaskVersion)</SdkVersionForWorkloadTesting> -->
+    <SdkVersionForWorkloadTesting>9.0.105</SdkVersionForWorkloadTesting>
```

Comment:
@akoeplinger Should I merge this in to avoid non-existent `9.0.106` or do we handle it separately?